SecretSanta/SecretSanta.Business.Tester/GiftTests.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections;
+
+namespace SecretSanta.Business.Tester
+{
+    [TestClass]
+    public class GiftTests
+    {
+        [TestMethod]
+#pragma warning disable CA1707 // Identifiers should not contain underscores
+        public void Create_Gift_Success()
+#pragma warning restore CA1707 // Identifiers should not contain underscores comment: Unit Test are okay
+        {
+            //arrange
+            ArrayList gifts = new ArrayList();
+            User user = new User(2, "John", "Doe", gifts);
+            int Id = 5;
```

Comment:
```suggestion
            int id = 5;
```
camelCase for fields